.github/workflows/test_fast.yml

```
@@ -18,7 +23,14 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Apt-Get
+      - name: Brew Install
+        if: startsWith(matrix.os, 'macos')
+        run: |
+          brew update
+          brew install shellcheck sqlite3
+
+      - name: Apt-Get Install
+        if: endsWith(matrix.os, 'ubuntu')
```

I think this is meant to be?
```suggestion
        if: startsWith(matrix.os, 'ubuntu')
```

---

Crud, well spotted.

---

Amusingly the tests all pass since the things we are installing are already in the base environment.